simpleFit.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python
+# =============================================================================
+# @file   simpleFit.py
+# @author C. Marin Benito (carla.marin.benito@cern.ch)
+# @date   22.12.2014
+# =============================================================================
+"""This script fits a simple pdf to a given dataset using RooFit."""
+
+# imports
+import os
+import ROOT
+from uncertainties import ufloat
+
+RooFit         = ROOT.RooFit
+RooRealVar     = ROOT.RooRealVar
+RooArgList     = ROOT.RooArgList
+RooArgSet      = ROOT.RooArgSet
+RooDataSet     = ROOT.RooDataSet
+RooGaussian    = ROOT.RooGaussian
+RooExponential = ROOT.RooExponential
+RooAddPdf      = ROOT.RooAddPdf
+
+# definition of functions for this script
+def simpleFit(tree, cuts, mean, xmin = 4000, xmax = 7000):
+    """
+    This function fits the "B_M" variable of a given TTree
+    with a model formed by a Gaussian and an exponential pdf.
+    All shape parameters are allowed to float in the fit. The 
+    initial and range values are hardcoded in the code, except
+    for the initial value of the Gaussian mean and the range
+    of the B_M variable to be used.
+    Returns the composed model (RooAbsPdf), the residual plot object
+    and its chi2 value (float)
+    Definition of the arguments:
+    :tree: type TTree
+    the root TTree that contains the variable to be fitted
+    :cuts: type str
+    optional cuts to apply to the TTree before fitting
+    :mean: type float
+    initial value for the Gaussian mean that will be floated
+    during the fit
+    :xmin: type float, optional
+    minimum value of the B_M range to be fitted. Default: 4000
+    :xmax: type float, optional
+    maximum value of the B_M range to be fitted. Default: 7000
+    """
+    
+    # define variables and pdfs
+    B_M = RooRealVar("B_M","B_M", xmin, xmax)
```

Comment:
this needs to be changed to use the name of the mass variable we want to fit as it appears in our ntuples, ie:
```suggestion
    Lb_M = RooRealVar("Lb_M","Lb_M", xmin, xmax)
```
(please check, I'm writing this from memory). Also all the places where the `B_M` variable is used, it should be changed to `Lb_M`